install.sh

```diff
@@ -295,21 +294,41 @@ test_git() {
   version_ge "$(major_minor "${git_version_output##* }")" "$(major_minor "$REQUIRED_GIT_VERSION")"
 }
 
+# Search given executable in all PATH entries (remove dependency for `which` command)
+which_all() {
+  if [[ $# -ne 1 ]]; then
+    return 1
+  fi
+
+  local executable entries entry presented=''
+  IFS=':' read -r -a entries <<< "${PATH}"  # `readarray -d ':' -t` seems not applicable on WSL Bash
+  for entry in "${entries[@]}"; do
+    executable="${entry}/$1"
+    if [[ -x "${executable}" ]]; then
+      echo "${executable}"
+      presented='yes'
+    fi
+  done
+  if [[ -n "${presented}" ]]; then
```

Comment:
```suggestion
  local executable entries entry present=''
  IFS=':' read -r -a entries <<< "${PATH}"  # `readarray -d ':' -t` seems not applicable on WSL Bash
  for entry in "${entries[@]}"; do
    executable="${entry}/$1"
    if [[ -x "${executable}" ]]; then
      echo "${executable}"
      present='yes'
    fi
  done
  if [[ -n "${present}" ]]; then
```
Or, maybe use `available` instead of `present`.